src/components/IdentifierCardEditing.vue

```diff
@@ -113,20 +116,63 @@ export default defineComponent({
             default: 0
         }
     },
+    components: {
+        SchemaGuideLink
+    },
     setup (props) {
+        const linkInfo: linkInfoType = {
+            doi: { linkLabel: 'DOI', linkUrl: '#definitionsdoi' },
+            url: { linkLabel: 'URL', linkUrl: '#definitionsurl' },
+            swh: {
+                linkLabel: 'Software Heritage identifier',
+                linkUrl: '#definitionsswh-identifier'
+            },
+            other: { linkLabel: 'identifier', linkUrl: '#definitionsidentifier-description' }
+        }
+
         return {
             typeOptions: [
                 { label: 'DOI', value: 'doi' },
                 { label: 'URL', value: 'url' },
                 { label: 'Software Heritage', value: 'swh' },
                 { label: 'Other', value: 'other' }
             ],
+            getLinkLabel: computed(() => {
+                // console.log("label props:", props.type);
+                // return "Label"
+                return linkInfo[props.type].linkLabel
+            }),
+            getLinkUrl: computed(() => {
+                // let reallyString = <string>props.type;
+                // console.log("url props:", props.type);
+                // console.log("reallyString:", reallyString);
+                // return "Url"
+                return linkInfo[props.type].linkUrl
+            }),
             typeError: computed(() => getMyErrors(`/identifiers/${props.index}/type`)),
             valueError: computed(() => getMyErrors(`/identifiers/${props.index}/value`)),
-            descriptionError: computed(() => getMyErrors(`/identifiers/${props.index}/description`)),
-            identifierErrors: computed(() => identifierErrors(props.index))
+            descriptionError: computed(() =>
+                getMyErrors(`/identifiers/${props.index}/description`)
+            ),
+            identifierErrors: computed(() => identifierErrors(props.index)),
+            SchemaGuideLink
```

Comment:
dont think we need `SchemaGuideLink` here

Reply:
@fdiblen, I think you missed this one, and the comments in https://github.com/citation-file-format/cff-initializer-javascript/pull/495#pullrequestreview-975264920